src/App.jsx

```diff
@@ -63,6 +76,15 @@ function App() {
     fetchData();
   }, []);
 
+  // Display reset level button if completed todos reaches 16
+  useEffect(() => {
+    if (completedTodosCount - 1 >= 15) {
+      setShowResetLevelButton(true);
+    } else {
+      setShowResetLevelButton(false);
+    }
+  }, [completedTodosCount]); 
+
   // Update local storage with most recent todoList
   useEffect(() => {
     if (!isLoading) {
```

Comment:
You should be able to remove this `localStorage` usage from where the todos are now stored in Airtable. There is an argument to be made for having a local backup, though.